packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2817,6 +2818,14 @@ export class ContainerRuntime
 				forcedFullTree,
 			} as const;
```

Comment:
nit: Add a type assertion `const generateSummaryData: IGenerateSummaryTreeResult = ...` since we don't have the type checking in the return type (since `stage` is `base` now)

Reply:
Added type omitting stage and error which are added when returning it. Take a look.